doc/rados/troubleshooting/troubleshooting-mon.rst

```diff
@@ -383,6 +383,80 @@ example::
 
 	iptables -A INPUT -m multiport -p tcp -s {ip-address}/{netmask} --dports 6789,6800:7300 -j ACCEPT
 
+Monitor Store Failures
+======================
+
+Symptoms of store corruption
+----------------------------
+
+Ceph monitor stores the `cluster map`_ in a key/value store. If a monitor in a
```

Comment:
We might want to say something like "a key/value store such as LevelDB".

I realize it's configurable and we might switch to RocksDB, but I think in the medium term anybody looking at this is going to know that LevelDB got messed up and not seeing that this is a repair tool for that case will confuse them. (Avoiding the name is especially weird since we are including LevelDB error messages below.)